docs/bin/plugin_formatter.py

```diff
@@ -64,7 +64,7 @@ def html_escape(text, quote=True):
 
 # if a module is added in a version of Ansible older than this, don't print the version added information
 # in the module documentation because everyone is assumed to be running something newer than this already.
-TOO_OLD_TO_BE_NOTABLE = 1.3
+TOO_OLD_TO_BE_NOTABLE = 2.0
```

Comment:
note: in the future this should just be 'distance from current' instead of a specific version number to avoid having to update manually

Reply:
@bcoca good idea. I think we can merge this one for now and I'll add this to my docs-checklist-for-new-releases.

Reply:
Added to docs checklist, hope we can implement the "distance from current" thing with the release of 2.8.

Reply:
Personally I think it's better to do this manually. It won't be easy to calculate over major releases and the most important metric is whether people are actively using older versions.

Not sure how we could get proper metrics related to this though, except maybe through customer support/sales.